staging/kos/pkg/registry/network/subnet/strategy.go

```diff
@@ -33,13 +32,15 @@ import (
 	"k8s.io/examples/staging/kos/pkg/apis/network"
 	informers "k8s.io/examples/staging/kos/pkg/client/informers/internalversion"
 	listers "k8s.io/examples/staging/kos/pkg/client/listers/network/internalversion"
+	"k8s.io/examples/staging/kos/pkg/util/parse/network/subnet"
 )
 
 // NewStrategy creates a subnetStrategy instance and returns a pointer to it.
-func NewStrategy(typer runtime.ObjectTyper, listerFactory informers.SharedInformerFactory) *subnetStrategy {
+func NewStrategy(typer runtime.ObjectTyper, checkConflicts bool, listerFactory informers.SharedInformerFactory) *subnetStrategy {
```

Comment:
I suggest using the SharedIndexInformer here to maintain an index on VNI, making it cheaper to look for conflicts.

Reply:
Fixed in 5254ee4.